common/views/components/Dot/Dot.tsx

```diff
@@ -16,7 +17,7 @@ type Props = {
 };
 
 const Dot: FunctionComponent<Props> = ({ color }: Props) => {
-  return <DotEl className={`font-${color}`} />;
+  return <DotEl className={classNames({ [`font-${color}`]: true })} />;
```

Comment:
Added checks in `classNames` to transform `accent.green` to `accent-green` as dots in class names are no nos.